demisto_sdk/commands/download/downloader.py

```diff
@@ -604,7 +604,8 @@ def file_type_to_entity(file_data: dict, file_type: str) -> str:
         """
         if file_type and file_type == 'playbook':
             name: str = get_entity_name_by_entity_type(file_data, PLAYBOOKS_DIR)
-            if name and 'test' in name.lower():
+            if re.match(r'\btest\b', name.lower()) or name.endswith(("Test", "_test", "_Test")) \
```

Comment:
the `\btest\b` isn't documented under the command's `README`, is it redundant or just missing from the docs? Please make sure they match.